client/components/post-schedule/index.jsx

```diff
@@ -62,7 +62,7 @@ class PostSchedule extends Component {
 		showTooltip: false,
 	};
 
-	componentWillMount() {
+	UNSAFE_componentWillMount() {
```

Comment:
An example of how this might look as a property initializer:
```js
state = this.props.selectedDay
  ? {
    localizedDate: null,
	isFutureDate: false,
  }
  : {
    localizedDate: this.getDateToUserLocation( this.props.selectedDay ), // not sure if this will actually work though
    isFutureDate: this.getDateToUserLocation( this.props.selectedDay ).isAfter(), // actually, we could do better than calling this twice
  }

Reply:
As discussed we will defer this for another PR!